inventory/byo/hosts.ose.example

```diff
@@ -167,6 +167,14 @@ openshift_release=v3.6
 # modify image streams to point at that registry by setting the following to true
 #openshift_examples_modify_imagestreams=true
 
+# If oreg_url points to a registry requiring authentication, provide the following:
+oreg_host: example.com:443
+registry_auth_user: some_user
+registry_auth_token: 'my-token'
```

Comment:
Should not we get oreg_host from oreg_url? Seem like oreg_host is a little redundant.